### PR TITLE
Brands API: Deprecate page from VuePress

### PR DESCRIPTION
**Pull request by brentschaus-okta**
## Description:
- **What's changed?** Deprecates Brands API page from VuePress
- **Is this PR related to a Monolith release?** No.

### Resolves:

* [OKTA-737799](https://oktainc.atlassian.net/browse/OKTA-737799)

### CODE REVIEW

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/custom-email/main/index.md](https://github.com/okta/okta-developer-docs/blob/99a5c446a6c5ed6149d3c16f6734354797cad6b9/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fcustom-email%2Fmain%2Findex.md) | [80](https://okta.acrolinx.cloud/api/v1/checking/scorecards/7a6758fc-b837-4a02-b225-b93b5673aafc) | 70  | 90  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/custom-error-pages/main/index.md](https://github.com/okta/okta-developer-docs/blob/99a5c446a6c5ed6149d3c16f6734354797cad6b9/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fcustom-error-pages%2Fmain%2Findex.md) | [87](https://okta.acrolinx.cloud/api/v1/checking/scorecards/56ce14e1-f608-483c-a748-76a552771d3d) | 72  | 80  | 80  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/terraform-manage-end-user-experience/main/index.md](https://github.com/okta/okta-developer-docs/blob/99a5c446a6c5ed6149d3c16f6734354797cad6b9/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fterraform-manage-end-user-experience%2Fmain%2Findex.md) | [86](https://okta.acrolinx.cloud/api/v1/checking/scorecards/897331d2-c668-4852-818c-97966054e72b) | 82  | 73  | 93  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.010ae86f-35ff-42c5-b2e9-ddf87b0f320c?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/custom-email/main/index.md](https://github.com/okta/okta-developer-docs/blob/1a0c3a5faf996281368c1ee0ad0b382dea13e209/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fcustom-email%2Fmain%2Findex.md) | [80](https://okta.acrolinx.cloud/api/v1/checking/scorecards/f7fab72f-818c-4682-8f90-276401aa4b65) | 70  | 90  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/custom-error-pages/main/index.md](https://github.com/okta/okta-developer-docs/blob/1a0c3a5faf996281368c1ee0ad0b382dea13e209/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fcustom-error-pages%2Fmain%2Findex.md) | [87](https://okta.acrolinx.cloud/api/v1/checking/scorecards/b904a512-c02e-4564-9437-e1abfd18960e) | 72  | 80  | 80  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/terraform-manage-end-user-experience/main/index.md](https://github.com/okta/okta-developer-docs/blob/1a0c3a5faf996281368c1ee0ad0b382dea13e209/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fterraform-manage-end-user-experience%2Fmain%2Findex.md) | [86](https://okta.acrolinx.cloud/api/v1/checking/scorecards/9b673e10-3eee-4466-8f25-a39a3ca3be62) | 82  | 73  | 93  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.3e4bd6fe-c0d0-4f24-a7a6-b1bef5225260?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/custom-email/main/index.md](https://github.com/okta/okta-developer-docs/blob/6994561c00085b8a6a5508bfdfbe3916e5e8368b/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fcustom-email%2Fmain%2Findex.md) | [80](https://okta.acrolinx.cloud/api/v1/checking/scorecards/51da4b27-f5c6-4db0-9a31-9837c0ef2ea8) | 70  | 90  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/custom-error-pages/main/index.md](https://github.com/okta/okta-developer-docs/blob/6994561c00085b8a6a5508bfdfbe3916e5e8368b/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fcustom-error-pages%2Fmain%2Findex.md) | [87](https://okta.acrolinx.cloud/api/v1/checking/scorecards/bbf182dd-1a6b-4143-a1cd-fb6466e93f2e) | 72  | 80  | 80  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/terraform-manage-end-user-experience/main/index.md](https://github.com/okta/okta-developer-docs/blob/6994561c00085b8a6a5508bfdfbe3916e5e8368b/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fterraform-manage-end-user-experience%2Fmain%2Findex.md) | [86](https://okta.acrolinx.cloud/api/v1/checking/scorecards/2ae2dce6-85e6-4372-847b-0074ccd0ecde) | 82  | 73  | 93  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.be1b6c9d-b0a2-43f8-9bd1-9fdf2952f488?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

File: packages/@okta/vuepress-site/docs/release-notes/2021/index.md [hunk 1]
```diff
@@ -242,7 +242,7 @@ You can now configure your org to use the Native SSO flow. The [Native SSO](/doc
 
 #### New Brands API option to remove sign-in page footer message
 
-You can now remove "Powered by Okta" and "© 2021" from the Okta-hosted sign-in page using either the [Customizations > Footer](https://help.okta.com/okta_help.htm?id=ext-custom-footer) option in the Admin Console or the [Brands API](/docs/reference/api/brands/#brand-api-objects). <!--OKTA-424736-->
+You can now remove "Powered by Okta" and "© 2021" from the Okta-hosted sign-in page using either the [Customizations > Footer](https://help.okta.com/okta_help.htm?id=ext-custom-footer) option in the Admin Console or the [Brands API](https://developer.okta.com/docs/api/openapi/okta-management/management/tag/Brands/#tag/Brands/operation/replaceBrand). <!--OKTA-424736-->
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2021/index.md:245]:
```suggestion
You can now remove "Powered by Okta" and "© 2021" from the Okta-hosted sign-in page using either the [**Customizations** > **Footer**](https://help.okta.com/okta_help.htm?id=ext-custom-footer) option in the Admin Console or the [Brands API](https://developer.okta.com/docs/api/openapi/okta-management/management/tag/Brands/#tag/Brands/operation/replaceBrand). <!--OKTA-424736-->
```

**Review by susanharper-okta — COMMENTED** [review]
1 comment

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/custom-email/main/index.md](https://github.com/okta/okta-developer-docs/blob/261121d44fa833aaa3319630803899ab7ec28df6/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fcustom-email%2Fmain%2Findex.md) | [80](https://okta.acrolinx.cloud/api/v1/checking/scorecards/07ecc6a0-5b0c-40f0-a03d-ee2cce46f3e5) | 70  | 90  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/custom-error-pages/main/index.md](https://github.com/okta/okta-developer-docs/blob/261121d44fa833aaa3319630803899ab7ec28df6/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fcustom-error-pages%2Fmain%2Findex.md) | [87](https://okta.acrolinx.cloud/api/v1/checking/scorecards/c6681556-8e17-41e6-bdc9-e6ad7e840bd8) | 72  | 80  | 80  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/terraform-manage-end-user-experience/main/index.md](https://github.com/okta/okta-developer-docs/blob/261121d44fa833aaa3319630803899ab7ec28df6/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fterraform-manage-end-user-experience%2Fmain%2Findex.md) | [86](https://okta.acrolinx.cloud/api/v1/checking/scorecards/a6c5e01f-7204-42e8-98b4-ea20ed3f1831) | 82  | 73  | 93  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.d0d3bfc5-1d87-4245-8c49-eaaa5e255d2f?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Review by susanharper-okta — APPROVED** [review]
LGTM

**Pull request merged**